imager_images/models.py

```diff
@@ -32,71 +35,42 @@ class Photo(models.Model):
     title = models.CharField("Title", name='title', max_length=255, blank=True)
     height_field = models.IntegerField("Height", blank=True)
     width_field = models.IntegerField("Width", blank=True)
-    latitude = models.IntegerField("Latitude", blank=True)
-    longitude = models.IntegerField("Longitude", blank=True)
-    camera = models.CharField("Camera", max_length=64, blank=True)
-    lens = models.CharField("Lens", max_length=64, blank=True)
-    focal_length = models.CharField("Focal Length", max_length=32, blank=True)
-    shutter_speed = models.IntegerField("Shutter Speed", blank=True)
-    appature = models.CharField("Title", max_length=64, blank=True)
-    description = models.CharField("Title", max_length=255, blank=True)
+    description = models.CharField("Description", max_length=255, blank=True, null=True)
     date_created = models.DateField('Date Created', auto_now_add=True)
     date_modified = models.DateField('Date Modified', auto_now=True)
-    date_pub = models.DateField('Date Published', editable=True, blank=True)
+    date_pub = models.DateField('Date Published', editable=True, blank=True, null=True)
     published = models.CharField(max_length=64,
                                  choices=[('private', 'private'),
                                           ('shared', 'shared'),
-                                          ('public', 'public')])
-    likes_cheese = models.BooleanField('Likes Cheese!', default=False)
+                                          ('public', 'public')],
+                                 default='private')
 
     def __str__(self):
         '''this is a  doc string'''
-        return self.title
+        return '{}: {}'.format(self.photographer.username, self.title)
```

Comment:
This is confusing. Why would the string representation of this object be the user that owns it and the title of the picture? That's not very descriptive...